swagger.yml

```diff
@@ -0,0 +1,190 @@
+swagger: "2.0"
+info:
+  title: Tigerblood
+  description: IP Reputation Service API
+  version: "1.0.0"
+  license:
+    name: MPL-2.0
+    url: http://mozilla.org/MPL/2.0/
+basePath: /
+schemes:
+- http
```

Comment:
Correction, I don't think we should remove http support, as tigerblood by itself does support it. We just have nginx on top of it, which is what disallows http.